src/api/department.ts

```diff
@@ -2,3 +2,48 @@ import express from "express";
 
 const router = express.Router();
 export default router;
+
+router.get("/department", (req, res) => {
+  res.send([
+    {
+      name: "Computer Science",
+      department_id: "CSC",
+    },
+    {
+      name: "Computer Engineering",
+      department_id: "CPE",
+    },
+  ]);
+});
+
+router.get("/department/:department_id", (req, res) => {
```

Comment:
We should keep URL params consistent, and use pascalCase instead of snake_case

Reply:
This is also something we kinda overlooked when writing the API spec, since we originally thought we were gonna do this in python, probably fine for now, once we're doing I can make another commit that goes back and fixes everything